website_docs/manual_instrumentation.md

```diff
@@ -443,41 +457,33 @@ Other exporters can be found in the [OpenTelemetry Registry].
 
 ### Auto Configuration
 
-To configure the OpenTelemetry SDK based on the standard set of environment variables and system
-properties, you can use the `opentelemetry-sdk-extension-autoconfigure` module.
+To configure the OpenTelemetry SDK based on the standard set of environment
+variables and system properties, you can use the
+`opentelemetry-sdk-extension-autoconfigure` module.
 
 ```java
   OpenTelemetrySdk sdk = OpenTelemetrySdkAutoConfiguration.initialize();
```

Comment:
Here an elsewhere, Java code shouldn't be indented unless the indentation is significant, which, in this case, and most other cases I've seen, it doesn't seem to be.

I've opened https://github.com/open-telemetry/opentelemetry.io/issues/996 to track this, if you agree @cartermp